### ISSUE TITLE
Simple check for unbalanced tuple unpacking in assignments

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#37]
Originally reported by: **Claudiu Popa (BitBucket: [PCManticore](http://bitbucket.org/PCManticore), GitHub: @PCManticore)**

---

Hello!

The following bug bit me in a production environment and I was surprised that Pylint didn't warn me about it
After fiddling with checkers and astroid nodes, I've managed to pull off a small patch which reports this kind of problems. I took in account only tuple unpacking and handled only the assignments, so the patch could  be small enough to be accepted. Also, I'm not a native english speaker and the wording in the patch can be improved.
Thank you for your time!

```
#!python

   a, b = "something1", "something2", "something3" # fails

```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/37
